src/nbt.rs

```diff
@@ -26,16 +31,14 @@ impl Nbt {
             return Err(Error::NoRootCompound(tag_type_id));
         }
 
-        let compound_name = get_nbt_string(bytes)?;
-
         Ok(Nbt {
-            name: compound_name,
-            root_tag: NbtCompound::deserialize(bytes),
+            name: get_nbt_string(bytes)?,
+            root_tag: NbtCompound::deserialize_content(bytes)?,
         })
     }
 
-    /// Reads Nbt tag, that doesn't contain the name of root compound
-    /// Used in [Network NBT](https://wiki.vg/NBT#Network_NBT_(Java_Edition))
+    /// Reads Nbt tag, that doesn't contain the name of root compound.
+    /// Used in [Network NBT](https://wiki.vg/NBT#Network_NBT_(Java_Edition)).
```

Comment:
Disparity in comments `NBT` and `Nbt` are we refering to Nbt object? if so clarify this